channel.go

```diff
@@ -43,6 +43,9 @@ var (
 
 const (
 	ephemeralHostPort = "0.0.0.0:0"
+
+	// DefaultTraceSampleRate is the default sampling rate for traces.
+	DefaultTraceSampleRate = 1.0
```

Comment:
This is the probability, correct?  Can we make the default to be `0.01` (1pct), as in Node implementation?